src/app/shared/log-in/password/log-in.component.html

```diff
@@ -17,9 +18,15 @@
          formControlName="password"
          required
          type="password">
-  <div *ngIf="(error | async) && hasError" class="alert alert-danger" role="alert" @fadeOut>{{ (error | async) | translate }}</div>
-  <div *ngIf="(message | async) && hasMessage" class="alert alert-info" role="alert" @fadeOut>{{ (message | async) | translate }}</div>
-  <button class="btn btn-lg btn-primary btn-block mt-3" type="submit" [disabled]="!form.valid">{{"login.form.submit" | translate}}</button>
+  <div *ngIf="(error | async) && hasError" class="alert alert-danger" role="alert"
+       @fadeOut>{{ (error | async) | translate }}</div>
+  <div *ngIf="(message | async) && hasMessage" class="alert alert-info" role="alert"
+       @fadeOut>{{ (message | async) | translate }}</div>
+
+  <button class="btn btn-lg btn-primary btn-block mt-3" type="submit"
+          [disabled]="!form.valid">{{"login.form.submit" | translate}}</button>
+
+
   <div class="dropdown-divider"></div>
```

Comment:
the password login component shoul contain only form and submit button, I should move all the rest to the login container

Reply:
Did you mean to make the log-in.component a presentational component?